lib/super_diff/rspec/differs/collection_including.rb

```diff
@@ -3,7 +3,8 @@ module RSpec
     module Differs
       class CollectionIncluding < SuperDiff::Differs::Array
         def self.applies_to?(expected, actual)
-          SuperDiff::RSpec.a_collection_including_something?(expected) &&
+          (SuperDiff::RSpec.a_collection_including_something?(expected) ||
```

Comment:
Nitpick: What do you think about formatting it like this so the grouping is easier to parse?

``` ruby
(
  SuperDiff::RSpec.a_collection_including_something?(expected) ||
  SuperDiff::RSpec.array_including_something?(expected)
) && actual.is_a?(::Array)
```

or 

``` ruby
(
  SuperDiff::RSpec.a_collection_including_something?(expected) ||
  SuperDiff::RSpec.array_including_something?(expected)
) &&
actual.is_a?(::Array)
```

Reply:
Makes sense. Fixed https://github.com/mcmire/super_diff/compare/c98b483c0c8d7787f1eb0d57de913ce12849b370..60938bd4a380d87b075dbdc3ff7e047085a1ebdd